src/App/Fossa/Config/Container.hs

```diff
@@ -122,12 +100,9 @@ instance GetCommonOpts ContainerCommand where
   getCommonOpts = \case
     ContainerAnalyze (ContainerAnalyzeOptions{analyzeCommons}) -> Just analyzeCommons
     ContainerTest (ContainerTestOptions{testCommons}) -> Just testCommons
-    ContainerParseFile _ -> Nothing
-    ContainerDumpScan _ -> Nothing
     ContainerListTargets _ -> Nothing
 
 parser :: Parser ContainerCommand
-parser = public <|> private
+parser = public
```

Comment:
[no action]: Good choice on not inlining `public` here.  I think that helps us track where we use public and private commands in other supercommands, so I like keeping the pattern going here.